_topic_maps/_topic_map.yml

```diff
@@ -2603,6 +2601,15 @@ Topics:
 - Name: Exported fields
   File: cluster-logging-exported-fields
   Distros: openshift-enterprise,openshift-origin
+- Name: API reference
+  Dir: api_reference
+  Topics:
+#  - Name: 5.8 Logging API reference
+#    File: logging-5-8-reference
+#  - Name: 5.7 Logging API reference
+#    File: logging-5-7-reference
```

Comment:
General comment for each of the topic maps. Just want to verify that you intended on stubbing out 5.8 and 5.7 from the TOC for now.

Reply:
Yeah that was the idea, just basically wanting to make a placeholder space for these at the moment. Not a big deal to remove them for now either though if this isn't OK to do?

Reply:
> Yeah that was the idea, just basically wanting to make a placeholder space for these at the moment. Not a big deal to remove them for now either though if this isn't OK to do?

Totally fine in my book to leave them stubbed in. I just wanted to make sure that you did in fact want to leave them commented out.